app/views/admin/organisations/_form.html.erb

```diff
@@ -49,5 +49,14 @@
         </div>
       <% end %>
     </div>
+    <div class="form-group">
```

Comment:
This should not be displayed when an organisation has no adapter at all - in such case selecting it and submitting the form causes an exception.